ui/lib/kv/addon/components/kv-patch/subkeys-reveal.hbs

```diff
@@ -8,6 +8,6 @@
     <Hds::Text::Body @tag="p" @weight="semibold">Reveal subkeys in JSON</Hds::Text::Body>
   </Toggle>
   {{#if this.showSubkeys}}
-    <Hds::CodeBlock @value={{stringify @subkeys}} @hasLineNumbers={{false}} data-test-subkeys />
+    <Hds::CodeBlock @value={{stringify @subkeys}} @hasLineNumbers={{false}} data-test-code-block="subkeys" />
```

Comment:
I updated all selectors added to the `Hds::CodeBlock` component so that we consistently used one GENERAL selector.